src/Teams.Tests/Domain/AnswerTests.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using Teams.Domain;
+
+namespace Teams.Tests.Domain
+{
+    public class AnswerTests
+    {
+        private Guid _predefinedId;
+        private Answer _defaultAnswerAsText;
+        private Answer _defaultAnswerAsOptions;
+
+        [SetUp]
+        public void Setup()
+        {
+            _predefinedId = new Guid("f4340761-1211-48eb-9bea-b6052c28bac3");
+            _defaultAnswerAsText = new Answer(Guid.NewGuid(), "answer");
+            _defaultAnswerAsOptions = new Answer(Guid.NewGuid(), GenerateFakeAnswersIds(1));
+        }
+
+        #region Generate_Answer_Ids
```

Comment:
Do not use regions for such small files and this is all coupled. Why do u splitted it?